SparkPost/matchbox
# Resolve issue #218 by adding "title" prop to Table component

Resolves #218 

## What Changed

- Added new `title` prop to `<Table/>` component to render `<caption>`
- This new prop is required, and as a result, represents a breaking change for applications that leverage it

## How to Test

- Go to: http://localhost:9001/?selectedKind=Layout%7CTable&selectedStory=table%20components&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel and verify that the `<caption></caption>` renders
- Additionally, verify that the caption does not render visually
